### ISSUE TITLE
How to debug skflow?

### ISSUE THREAD

**Issue by mahatosourav91** [wookayin/tensorflow-talk-debugging#5]
Hi I am new to tensorflow. I want to debug Tensorflow (skflow) gmm_ops.py (Gaussian Mixture Model). I am   getting ERROR:tensorflow:Model diverged with loss = NaN.
 How should I do it ? Is there any example?
